src/main/java/alfred/exception/InvalidDateException.java

```diff
@@ -0,0 +1,16 @@
+package alfred.exception;
+
+/**
+ * Represents errors in user input for invalid date and time.
+ */
+public class InvalidDateException extends AlfredException {
+
+    private static final String INVALID_DATE = "Please try again with a valid date and time DD/MM/YYYY HHMM";
+
+    /**
+     * Creates a new exception object for invalid date.
+     */
+    public InvalidDateException() {
+        super(INVALID_DATE);
```

Comment:
Would it be possible to move the string into the super()
```suggestion
        super("Please try again with a valid date and time DD/MM/YYYY HHMM");
```